index.html

```diff
@@ -12,28 +12,72 @@
         <p class="card-text">How many items are in here?</p>
-        <button class="card-answer-btn" type="button">Show Answer</button>
+        <button class="card-answer-btn" type="button" aria-label="Click this button to show the answer of this question">Show Answer</button>
         <p class="answer">Answer</p>
         <span class="card-categories">
-          <a href="#">#html</a>
-          <a href="#">#css</a>
-          <a href="#">#javascript</a>
+          <i>#html</i>
+          <i>#css</i>
+          <i>#javascript</i>
         </span>
-      </div>
+      </article>
+      <article class="card">
+        <i class="fa-solid fa-bookmark"></i>
+        <p class="card-text">How many items are in here?</p>
+        <button class="card-answer-btn" type="button" aria-label="Click this button to show the answer of this question">Show Answer</button>
+        <p class="answer">Answer</p>
+        <span class="card-categories">
+          <i>#html</i>
+          <i>#css</i>
+          <i>#javascript</i>
+        </span>
+      </article>
+      <article class="card">
+        <i class="fa-solid fa-bookmark"></i>
+        <p class="card-text">How many items are in here?</p>
+        <button class="card-answer-btn" type="button" aria-label="Click this button to show the answer of this question">Show Answer</button>
+        <p class="answer">Answer</p>
+        <span class="card-categories">
+          <i>#html</i>
+          <i>#css</i>
+          <i>#javascript</i>
+        </span>
+      </article>
+      <article class="card">
+        <i class="fa-solid fa-bookmark"></i>
+        <p class="card-text">How many items are in here?</p>
+        <button class="card-answer-btn" type="button" aria-label="Click this button to show the answer of this question">Show Answer</button>
+        <p class="answer">Answer</p>
+        <span class="card-categories">
+          <i>#html</i>
+          <i>#css</i>
+          <i>#javascript</i>
+        </span>
+      </article>
+      <article class="card">
+        <i class="fa-solid fa-bookmark"></i>
+        <p class="card-text">How many items are in here?</p>
+        <button class="card-answer-btn" type="button" aria-label="Click this button to show the answer of this question">Show Answer</button>
+        <p class="answer">Answer</p>
+        <span class="card-categories">
+          <i>#html</i>
+          <i>#css</i>
+          <i>#javascript</i>
+        </span>
+      </article>
     </section>
   </main>
   <footer>
     <nav class="navigation-container">
-      <i class="fa-solid fa-home"></i>
-      <i class="fa-solid fa-bookmark"></i>
-      <i class="fa-solid fa-user"></i>
+      <a href="/" class="fa-solid fa-home active" title="This is the home to the home page"></a>
+      <a href="/bookmarks.html" class="fa-solid fa-bookmark" title="This navigates to your bookmarks"></a>
+      <a href="/profile.html" class="fa-solid fa-user" title="this navigates to your profile"></a>
```

Comment:
To make sure the links work properly in all occasions, better use relative paths:
```suggestion
      <a href="./bookmarks.html" class="fa-solid fa-bookmark" title="This navigates to your bookmarks"></a>
      <a href="./profile.html" class="fa-solid fa-user" title="this navigates to your profile"></a>
```
Besides, what's the difference between the `title` attribute and the `aria-label`?